modules/smtp/modules.php

```diff
@@ -1126,7 +1126,7 @@ protected function output() {
             smtp_server_dropdown($this->module_output(), $this, $recip, $selected_id).
             '<input class="smtp_send" type="submit" value="'.$this->trans('Send').'" name="smtp_send" '.$send_disabled.'/>';
 
-        if ($this->get('list_path') && $reply_type == 'reply') {
+        if ($this->get('list_path')) {
```

Comment:
Besides reply all, I think we have reply types as forward and maybe others. Are we sure we want this button everywhere?

Reply:
Hello @kroky. I pushed a new commit please check.